lib/ddtrace/ext/metrics.rb

```diff
@@ -0,0 +1,11 @@
+module Datadog
+  module Ext
+    module Metrics
+      TAG_LANG = 'language'.freeze
+      TAG_LANG_INTERPRETER = 'language-interpreter'.freeze
```

Comment:
No need to change anything here, but we typically do `snake_case` tags, `runtime-id` is the odd one out 🙄

Reply:
Oh, I thought the spec referred to `-` variables? If it's supposed to be `_` I can change that.

Reply:
... for now... like I said, no reason to change anything you have here, but think we are going to suggest a change to `runtime_id` in the future since `_` is more common with other tags we emit. 

Or to be little more explicit, `runtime_id` won't work since the backend expects `runtime-id`.